### PR TITLE
523 code of conduct

### LINKED ISSUE #523: Code of conduct
As a visitor to the site
So that I can see what Makers is all about
I would like to see a Code of Conduct

### PR DESCRIPTION
**Pull request by nikeshashar**
Closes #523 

Please could someone review and add any comments / changes to be made. There will be tickets made for future amendments.

### CODE REVIEW

File: source/code_of_conduct.html.haml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    %h2 Definitions
+    %h4 Harassment includes, but is not limited to:
+    %ol.default-ol
+      %li Offensive comments related to gender, gender identity and expression, sexual orientation, disability, mental illness, neuro(a)typicality, physical appearance, body size, race, age, regional discrimination, political or religious affiliation
+      %li Unwelcome comments regarding a person’s lifestyle choices and practices, including those related to food, health, parenting, drugs, and employment
+      %li Deliberate misgendering. This includes deadnaming or persistently using a pronoun that does not correctly reflect a person’s gender identity. You must address people by the name they give you when not addressing them by their username or handle
+      %li Physical contact and simulated physical contact (eg, textual descriptions like “<em>hug” or “backrub”)</em> without consent or after a request to stop
```
Comment (pitchinvasion) [inline source/code_of_conduct.html.haml]: Move the <em> outside of the quote mark

File: source/code_of_conduct.html.haml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    %h3 Diversity Statement
+    %p We encourage everyone to participate and are committed to building a community for all. Although we will fail at times, we seek to treat everyone both as fairly and equally as possible. Whenever a participant has made a mistake, we expect them to take responsibility for it. If someone has been harmed or offended, it is our responsibility to listen carefully and respectfully, and do our best to right the wrong.
+    %p Although this list cannot be exhaustive, we explicitly honor diversity in age, gender, gender identity or expression, culture, ethnicity, language, national origin, political beliefs, profession, race, religion, sexual orientation, socioeconomic status, and technical ability. We will not tolerate discrimination based on any of the protected characteristics above, including participants with disabilities.
+
+    %h3 Reporting Issues
+    %p If you experience or witness unacceptable behavior—or have any other concerns—please report it by contacting us via <em>contact@makersacademy.com</em>. All reports will be handled with discretion. In your report please include:
+    %ol.default-ol
+      %li Your contact information.
```
Comment (pitchinvasion) [inline source/code_of_conduct.html.haml]: mailing list archive/IRC logger is not something that applies to makers, maybe "email/slack post"

File: source/code_of_conduct.html.haml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      %li Names (real, nicknames, or pseudonyms) of any individuals involved. If there are additional witnesses, please include them as well. Your account of what occurred, and if you believe the incident is ongoing. If there is a publicly available record (e.g. a mailing list archive or a public IRC logger), please include a link.
+      %li Any additional information that may be helpful.
+      %p After filing a report, a representative will contact you personally, review the incident, follow up with any additional questions, and make a decision as to how to respond. If the person who is harassing you is part of the response team, they will recuse themselves from handling your incident. If the complaint originates from a member of the response team, it will be handled by a different member of the response team. We will respect confidentiality requests for the purpose of protecting victims of abuse.
+      %h3 Acknowledgements
+      %p This code of conduct is based on the Open Code of Conduct v1.0 from the TODOGroup. We are thankful for their work and all the communities who have paved the way with code of conducts.
```
Comment (pitchinvasion) [inline source/code_of_conduct.html.haml]: Should there be a link here?

**Comment by pitchinvasion** [comment]
Few minor questions otherwise :shipit:

File: source/code_of_conduct.html.haml (hunk 1 above)
Comment (nikeshashar) [inline source/code_of_conduct.html.haml]: @pitchinvasion do you mean move the " after backrub to outside the ending </em> ?

File: source/code_of_conduct.html.haml (hunk 3 above)
Comment (nikeshashar) [inline source/code_of_conduct.html.haml]: There will be in v1.1 @pitchinvasion - I'd like to get the basic version up asap.

**Pull request merged**